From 0a71181b81a5ba2e24dd812b998fc36cf2a31c79 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Wed, 3 Aug 2011 15:18:39 +0200
Subject: [PATCH 07/13] wizard: rename Configure Event -> Preferences,

- users don't know what Events are, so preferences sounds more user friendly
---
 src/gui-wizard-gtk/wizard.glade |    9 ++++++---
 1 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/src/gui-wizard-gtk/wizard.glade b/src/gui-wizard-gtk/wizard.glade
index c57b13d..076de96 100644
--- a/src/gui-wizard-gtk/wizard.glade
+++ b/src/gui-wizard-gtk/wizard.glade
@@ -210,12 +210,13 @@
             <property name="can_focus">False</property>
             <child>
               <object class="GtkButton" id="button_cfg1">
-                <property name="label" translatable="yes">Configure _Events</property>
+                <property name="label">gtk-preferences</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
                 <property name="use_action_appearance">False</property>
                 <property name="use_underline">True</property>
+                <property name="use_stock">True</property>
                 <property name="image_position">right</property>
               </object>
               <packing>
@@ -466,12 +468,13 @@
             <property name="can_focus">False</property>
             <child>
               <object class="GtkButton" id="button_cfg2">
-                <property name="label" translatable="yes">Configure _Events</property>
+                <property name="label">gtk-preferences</property>
                 <property name="visible">True</property>
                 <property name="can_focus">True</property>
                 <property name="receives_default">True</property>
                 <property name="use_action_appearance">False</property>
                 <property name="use_underline">True</property>
+                <property name="use_stock">True</property>
                 <property name="image_position">right</property>
               </object>
               <packing>
-- 
1.7.6

